### PR TITLE
feat: create incident screen

### PR DESCRIPTION
**Pull request by HavardNot**
image
image

### CODE REVIEW

**Comment by HavardNot** [comment]
This has not been tested properly as the ServiceNow API endpoint is not working in production, and Chronicles is only allowed to talk to the Production API. An issue to enable multi environment communication has been created here #303

File: packages/core/src/types.ts [hunk 1]
```diff
@@ -13,6 +13,12 @@ export type MadConfig = {
      * @see https://web-mad-service-portal-web-prod.radix.equinor.com/
      */
     servicePortalName: string;
+    /**
+     * ServiceNow Configuration Item (cmdb_ci) of the app.
+     * Will be used to create ServiceNow tickets for the correlating application.
+     * @see https://portal.azure.com/#@StatoilSRM.onmicrosoft.com/resource/subscriptions/2ad1b087-ffb8-4cf3-bcc2-8caeebfcd3f3/resourceGroups/mad-test/providers/Microsoft.Storage/storageAccounts/madtest62alhixjcodkm/storagebrowser
+     */
+    serviceNowConfigurationItem: string;
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply packages/core/src/types.ts:21]: Also, this replaces the older `serviceNow` config I added earlier. I suggest going through the files and see where that old one is used. I believe we check if it exists, and if it doesn't, we don't mount the screen to the app at all, and we don't add a navigation cell for the screen. Those parts of the code has to be changed to look at `serviceNowConfigurationItem` instead

[the remark this answers is not in the document]
Comment (HavardNot) [reply packages/core/src/types.ts]: renamed serviceNowConfigurationItem to serviceNow, made it optional and removed the old serviceNow config item

**Comment by HavardNot** [comment]
Working fine in Test environment. Still not verified to work in prod but this is an ongoing process which may be fixed in the back end. Ready for reeeeee-review 👍

**Pull request merged**